metadata-ingestion/src/datahub/ingestion/source/aws/glue.py

```diff
@@ -686,6 +687,7 @@ def get_all_databases(self) -> Iterable[Mapping[str, Any]]:
         # see https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/glue/paginator/GetDatabases.html
         paginator = self.glue_client.get_paginator("get_databases")
 
+        self.report.catalog_id = self.source_config.catalog_id
```

Comment:
Given this is a fixed value from the config, what's the value of having it in the report?

If valuable, could it be moved closer to the initialization of the `GlueSourceReport`? perhaps here https://github.com/datahub-project/datahub/blob/6bcc4469f9580a0729808f7c9f997275c3d1bc93/metadata-ingestion/src/datahub/ingestion/source/aws/glue.py#L318

Reply:
It is valuable because we don't always have a static report. Customers can be getting this from a parameter store and running it via python sdk

Reply:
Makes sense, thanks for explaining!

Not in the scope of this PR, this may be generalized to automatically include source/sink configs in the report, excluding/masking secrets, ofc